widget/select_test.go

```diff
@@ -66,17 +66,22 @@ func TestSelect_Tapped(t *testing.T) {
 }
 
 func TestSelect_Tapped_Constrained(t *testing.T) {
+	// fresh app for this test
```

Comment:
Is a new app needed? The NewWindow should be enough I thought?

Reply:
Again: no.
For the same reason as above: the code under test uses the driver of the current app for determining the canvas and this is short-circuited to the canvas of the first window of the app.

Reply:
But why is the app being tested no longer the current app?